packages/fxa-settings/src/components/FormPasswordWithBalloons/index.tsx

```diff
@@ -29,6 +29,8 @@ export type FormPasswordWithBalloonsProps = {
   disableButtonUntilValid?: boolean;
   submitButtonGleanId?: string;
   requirePasswordConfirmation?: boolean;
+  cmsButtonText?: string;
+  cmsButtonColor?: string;
```

Comment:
small non-blocking suggestion: maybe we could create a type with these cms properties and `&` it to the other types.  I see them repeated a bunch below.